### PR TITLE
Fix _3d-printing.scss

### PR DESCRIPTION
**Pull request by ortbogdan**
Checked classes on BEM.
Added font-family to model-examples__title.
Deleted unnecessary markup

### CODE REVIEW

File: src/sass/pages/_3d-printing.scss [hunk 1]
```diff
@@ -2,6 +2,7 @@
 .model-examples__title {
   margin-bottom: 16px;
 
+  font-family: Cera-Pro, sans-serif;
```
Comment (Zolotarova-Nataliia) [inline src/sass/pages/_3d-printing.scss:5]: We dont need this, because font-family added in _base.scss

Comment (ortbogdan) [reply src/sass/pages/_3d-printing.scss:5]: Modern-normalize break font-family styles of body

Comment (Androborn) [reply src/sass/pages/_3d-printing.scss:5]: Links order fixed. Please fix commit.

Comment (ortbogdan) [reply src/sass/pages/_3d-printing.scss]: Done

**Review by Androborn — APPROVED** [review]
Accepted

**Pull request merged**